src/compiler/checker.ts

```diff
@@ -3676,6 +3690,12 @@ module ts {
             return getTypeOfGlobalSymbol(getGlobalTypeSymbol(name), arity);
         }
 
+        function getNamespacedType(namespace: string, name: string): Type {

```

Comment:
getExportedTypeFromNamespace. Otherwise it sounds like the type of the namespace object.